src/rules/if_change_then_change.rs

```diff
@@ -8,7 +8,8 @@ use std::path::PathBuf;
 use regex::Regex;
 
 use crate::diagnostic;
-use crate::git::Hunk;
+use crate::git;
+use rayon::prelude::*;
```

Comment:
Nit. I usually avoid importing everything. This is essentially same as writing `using namespace std`

Reply:
Rayon doesn't seem to expose these in a more idiomatic manner that works correctly for the iterator traits. 

Open to suggestions on how this could be written better.